webapp/src/core.ts

```diff
@@ -347,8 +347,14 @@ export function promptAsync(options: PromptOptions): Promise<string> {
     options.onLoaded = () => {
         let dialogInput = document.getElementById('promptDialogInput') as HTMLInputElement;
         if (dialogInput) {
-            dialogInput.focus();
```

Comment:
Weirdly, without this line, the focus is still given to the input field (tested on Chrome, Edge, Firefox, Safari).

Furthermore, once the accessibility branch will be merged, the focus will still be given thanks to the initializeFocusTabIndex method. It works fine with or without this line, with or without the accessibility changes.